tests/entity.tests.js

```diff
@@ -2,6 +2,16 @@ describe("Space.domain.Entity", function() {
 
   let Entity = Space.domain.Entity;
 
+  it("extends Space.Struct", function() {
+    expect(Entity).to.extend(Space.Struct);
+  });
+
+  it('is serializable', function() {
```

Comment:
Since space:messaging is testing Ejsonable, it should be sufficient to just check for the [isSerializable flag](https://github.com/meteor-space/messaging/blob/develop/source/mixins/serializable.js#L25)